pontus_perception/pontus_perception/yolo/yolo_node.py

```diff
@@ -0,0 +1,84 @@
+import rclpy
+from rclpy.node import Node
+from sensor_msgs.msg import Image
+from pontus_msgs.msg import YOLOResultArray, YOLOResult
+import cv2
+from cv_bridge import CvBridge
+from ultralytics import YOLO
+from ament_index_python.packages import get_package_share_directory
+
+class YOLONode(Node):
+
+    def __init__(self):
+        super().__init__('perception_YOLO')
+
+        self.declare_parameters(namespace='', parameters=[
+            ('auv', '')
+        ])
+
+        pkg_share = get_package_share_directory('pontus_perception')
+
+        model_path = pkg_share + '/yolo/' + self.get_parameter('auv').value + '/model.pt'
+
+        self.cv_bridge = CvBridge()
+
+        self.model = YOLO(model_path)
+
+        self.threshold = 0.5
```

Comment:
Can you make this a parameter for the node instead of having it hard-coded? This parameter can be in a config file called `yolo.yaml`